src/main/java/recipeio/commands/FindCommand.java

```diff
@@ -0,0 +1,39 @@
+package recipeio.commands;
+import recipeio.recipe.Recipe;
+import recipeio.ui.UI;
+
+import java.util.ArrayList;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+
+public class FindCommand {
```

Comment:
Consider converting the magic strings into constants with meaningful names instead.